src/lib/tvgPictureImpl.h

```diff
@@ -101,6 +133,20 @@ struct Picture::Impl
         return true;
     }
 
+    bool size(uint32_t w, uint32_t h)
+    {
+        this->w = w;
+        this->h = h;
```

Comment:
we need a member flag to resize image in update()
resizing = true;

Reply:
added flag member